youtube_dl/extractor/polskieradio.py

```diff
@@ -15,6 +16,67 @@
 )
 
 
+class PolskieRadioProgrammeIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?polskieradio\.pl/(?P<id>\d+/\d+)'
```

Comment:
There is also another URL format: http://www.polskieradio.pl/7,Jedynka/5102,HISTORIA-ZYWA.